spec/rubocop/cop/style/file_name_spec.rb

```diff
@@ -162,7 +162,8 @@
 
       it 'registers an offense' do
         expect(cop.offenses.size).to eq(1)
-        expect(cop.messages).to eq(['Use snake_case for source file names.'])
+        expect(cop.messages).to eq(
```

Comment:
Pull this expectation to the first possible example in the file. That's our common convention for checking messages.